### ISSUE TITLE
mypy check python target dependency multi times

### ISSUE THREAD

**Issue by WeiYunKolmostar** [bazel-contrib/bazel-mypy-integration#66]
Hi, considering there was a case like
a.py is import by b.py and c.py, and there is 2 target b and c in BUILD file.
When running bazel-mypy on b and c, will a.py be checked 2 times, once in each target ?

**Comment by WeiYunKolmostar** [comment]
I found that in my repo, the python dependency is really complicated. If the underlying python files will be checked multi times, this will make the mypy checked really slow. Thanks for help me out.

**Comment by ewhauser** [comment]
This is a duplicate of: https://github.com/bazel-contrib/bazel-mypy-integration/issues/48

**Comment by adzenith** [comment]
Yeah, let's discuss in the other issue (#48). We should get the cache working to speed this up.
